pyramid_swagger/api.py

```diff
@@ -266,23 +266,49 @@ def _get_target_url(spec, target, current_path=''):
         )
 
 
+def _ensure_sane_key(fragment, mar_target):
+    '''
+    This function patches a nasty bug with the
+    'allOff' keyword in a spec, our resolved
```

Comment:
allOf calls _marshaled_target a bunch of times which is what causes the problem imo. I think it might also happen with additionalProperties and items, but I havent been able to reproduce it with them. Should I delete the comment?